CODEOWNERS

```diff
@@ -283,52 +299,60 @@ Kconfig*                                  @tejlmand
 /subsys/mpsl/                             @nrfconnect/ncs-dragoon
 /subsys/mpsl/cx/                          @ankuns @martintv
 /subsys/mpsl/fem/                         @ankuns @martintv
-/subsys/net/                              @rlubos
+/subsys/net/                              @nrfconnect/ncs-co-networking
 /subsys/net/lib/mqtt_helper/              @nrfconnect/ncs-cia
 /subsys/net/lib/azure_*                   @nrfconnect/ncs-cia @coderbyheart
 /subsys/net/lib/aws_*                     @nrfconnect/ncs-cia @coderbyheart
 /subsys/net/lib/ftp_client/               @MarkusLassila @tomi-font
 /subsys/net/lib/icalendar_parser/         @lats1980
-/subsys/net/lib/lwm2m_client_utils/       @rlubos @SeppoTakalo @juhaylinen
-/subsys/net/lib/nrf_cloud/                @plskeggs @jayteemo @glarsennordic
+/subsys/net/lib/lwm2m_client_utils/       @nrfconnect/ncs-co-networking @SeppoTakalo @juhaylinen
+/subsys/net/lib/nrf_cloud/		  @nrfconnect/ncs-nrf-cloud
 /subsys/net/lib/nrf_provisioning/         @SeppoTakalo @juhaylinen
 /subsys/net/lib/zzhc/                     @junqingzou
 /subsys/net/lib/wifi_credentials/         @nrfconnect/ncs-cia
 /subsys/net/lib/softap_wifi_provision/    @nrfconnect/ncs-cia
-/subsys/net/openthread/                   @rlubos @edmont @canisLupus1313 @maciejbaczmanski
-/subsys/nfc/                              @grochu @anangl
+/subsys/net/openthread/                   @nrfconnect/ncs-co-networking @edmont @canisLupus1313 @maciejbaczmanski
+/subsys/nfc/                              @grochu
 /subsys/nfc/rpc/                          @nrfconnect/ncs-protocols-serialization
+/subsys/nrf_compress/                     @nordicjm
 /subsys/nrf_rpc/                          @doki-nordic @KAGA164
 /subsys/partition_manager/                @hakonfam
```

Comment:
Not sure about this, but not me at least. 

Perhaps same as here? 

`/tests/subsys/partition_manager/static_pm_file/ @nordicjm @tejlmand`